phac-nml/irida-next
# Hide table and show message when a sample has 0 files

## What does this PR do and why?
When a sample has no files, shows a message instead of rendering an empty table.
Fixes #152

## Screenshots or screen recordings
Sample with an attachment
![image](https://github.com/phac-nml/irida-next/assets/16961365/14a7a4c6-5a59-4379-9175-6501d09fa842)

Sample without any attachments
![image](https://github.com/phac-nml/irida-next/assets/16961365/94f1c925-b029-49ab-b66d-41ffc5d34c11)

## How to set up and validate locally
1. pull branch, run, run tailwind to update ui
2. create a new project
3. create 2 samples
4. add an attachment to one of the samples
5. verify that the appropriate table or message is rendered depending on if the sample has attachments or not.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
